### ISSUE TITLE
Saving and opening files

### ISSUE THREAD

**Issue by wpfan1000** [phase1geo/Minder#195]
Hi,

Just have some feedback about how files are being saved and opened.

1)
If I save a file under a particular path using Save File As, and then repeat this, Minder does not seem to remember the file's path, instead the path will be /home. So then I have to reselect the path each time. IMHO this is not typical behaviour for any other program I have used. My feedback would be to have Minder remember the path of the current file in the Save As dialog.

2)
It seems Minder autosaves. If I open a file from a particular path, then Minder does not autosave the file in that path, but instead saves it in /home. IMHO this is not typical behaviour for any other program I have used.  I think Minder should save in the path of the file that the file had when it was opened.

3)
If Minder is closed and then I open a file by double clicking a .minder file from a particular path, and this file is the same file I worked on last in Minder, then Minder will open and it will have one tab with the same file because when Minder opens it remembers the last file used and there will be a 2nd tab with the same file, because I opened it. So there are now 2 copies of the same file open in Minder. This is confusing. 

I can then make changes to the same file in one tab and those changes are not reflected in the same file in the other tab.

What makes this more confusing to me is that I then have 2 copies of the same file open with one having changes and the other not, but they are both saving to the same file name? This is a conflict. Plus they are both saving not to the path of the file but to the file in /home?

One suggestion I would have is to not open the last used file or make this an option to open the last file used or not.

Personally I never open a program and then open a file in it. I always double click on a file to then open its associated program and have the file then open in the program. 

I never want a program to remember and open the last file I used. This is only useful if I only use one file always with one program but for me that is never the case.

Thanks for your consideration.

**Comment by phase1geo** [comment]
1. I think I can make this happen.

2. I have never seen Minder do this. The code will, of course, use the file name associated with the file to save back when auto-save occurs. This would be a bug if this could be demonstrated with a file in a single tab.

3. It is a bug that the same file can exist in two different tabs at the same time. I will get that fixed. As for the application remembering the state of the application (including last opened files), this is to conform the application to the elementary OS HIG. Since Minder is a native application for that platform, it must follow that user experience so I won’t change that aspect of the application.

**Comment by phase1geo** [comment]
I believe that #1 is now addressed in the 'devel' branch.  The save file existing in two tabs issue (#3) should also be fixed in the 'devel' branch.  Additionally, I fixed an issue where hitting the Shift-Control-S keyboard shortcut would bring up the "Open" dialog instead of the "Save As" dialog -- I don't know how long that issue has been in the code base.

All of these fixes should be generally available in the next stable release.
